final_task/rss_reader/items.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from dataclasses import dataclass
+from typing import List
+from feedparser import FeedParserDict
+from parser_rss import format_description
+
+
+@dataclass()
```

Comment:
You dont need brakets here, since you dont pass any arguments into this decorator

Reply:
fixed